### DISCUSSION TITLE
Import transactions from german ING bank account

### DISCUSSION THREAD

**Discussion by timaschew** [firefly-iii/firefly-iii#6389] [Firefly III]
When I try to connect Nordiget with my german ING bank account I'm redirect to myaccount.ing.com. On that page I can login with my 10 characters login and internet key, but when doing 2FA it's not working. I receive a popup on my mobile phone app, but I cannot open the screen to unlock the action. I've tried it 20 times, even refreshing (swiping in the app). I've tried to do a 2FA test when I create a transaction via the webinterface, then it works fine. Today I called the service and they mentioned something with proxy and that it won't work.

How do you import your transactions from you ING account?
I've tried to use the csv files, but since "Mandat" and "Referenz" is missing, it lead sometimes to duplicates, because of a identical description (and amount of course). Also some of my transactions have an empty "Verwendungszweck" because they use just "Referenz".

**Comment by JC5** [comment]
I'm sorry, is it something on the phone that's not working?

**Reply by timaschew** [reply]
Yes, kind of. So nordigen is redirecting me to myaccount.ing.com which seems to be a deprecated endpoint for third parties.
Usually the login happens via ing.de. I didn't find a place to contact nordigen about this, except this email: sales@nordigen.com 

I wonder how other people are doing it. ING bank in Germany gives the customer PDF or CSV files. PDF have good quality/data, but they are not machina readable. And the CSV filies have some issues which I've described in my first post. I think I will write a PDF to CSV coverter which will consider the `Mandat` and `Referenz` field and I will merge it into the description (`Verwendungszweck`). So it will be a drop in replacement.
